goalgorilla/open_social
# Issue #3225497 by Ressinel: Update the Lazy-load module from 2.0 to 3.10

## Problem
Update the Lazy-load module to have compatibility with Drupal 9

## Solution
Update to the latest version.

## Issue tracker
https://www.drupal.org/project/social/issues/3225497

## How to test
Before test need to ensure that we exactly don't need patches that were added by this [PR](https://github.com/goalgorilla/open_social/pull/1466). 
Necessary @ronaldtebrake consultation 😀
- [ ] Enable modules `social_lazy_loading` and `social_lazy_loading_images`
- [ ] Make additional settings if need on this page `/admin/config/content/lazy`
- [ ] Lazy loading functionality must work properly

## Screenshots
N/A

## Release notes
The Lazy-load module has been updated to 3.10. Please view the release notes from [3.x-dev](https://www.drupal.org/project/lazy/releases/8.x-3.x-dev), [3.0-beta1](https://www.drupal.org/project/lazy/releases/8.x-3.0-beta1), [3.0-beta2](https://www.drupal.org/project/lazy/releases/8.x-3.0-beta2), [3.0-beta3](https://www.drupal.org/project/lazy/releases/8.x-3.0-beta3), [3.0-beta4](https://www.drupal.org/project/lazy/releases/8.x-3.0-beta4), [3.0-beta5](https://www.drupal.org/project/lazy/releases/8.x-3.0-beta5), [3.0-beta6](https://www.drupal.org/project/lazy/releases/8.x-3.0-beta6), [3.0](https://www.drupal.org/project/lazy/releases/8.x-3.0), [3.1](https://www.drupal.org/project/lazy/releases/8.x-3.1), [3.2](https://www.drupal.org/project/lazy/releases/8.x-3.2), [3.3](https://www.drupal.org/project/lazy/releases/8.x-3.3), [3.4](https://www.drupal.org/project/lazy/releases/8.x-3.4), [3.5](https://www.drupal.org/project/lazy/releases/8.x-3.5), [3.6](https://www.drupal.org/project/lazy/releases/8.x-3.6), [3.7](https://www.drupal.org/project/lazy/releases/8.x-3.7), [3.8](https://www.drupal.org/project/lazy/releases/8.x-3.8), [3.9](https://www.drupal.org/project/lazy/releases/8.x-3.9), and [3.10](https://www.drupal.org/project/lazy/releases/8.x-3.10) of that module for the changes.

## Change Record
N/A

## Translations
N/A
